aries-site/src/data/wcag/components.json

```diff
@@ -47,6 +47,421 @@
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": "Minimum contrast ratio of 4.61 (selected state) does not exceed 7:1. But this will pass for larger text which requires 4.5:1."
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "needs discussion",
+        "notes": "What about edge case when user renders a video/audio content within a tab panel?"
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable",
+        "notes": "Need to clarify this."
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": "Minimum contrast ratio of 4.61 exceeds 3:1."
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "passed",
+        "notes": "Hover and focus states are indicated by background color and border."
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "passed"
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "passed"
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "passed"
+      },
+      {
+        "rule": "2.1.4",
+        "label": "Character Key Shortcuts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "needs discussion",
+        "notes": "What about edge case when user renders a video/audio content within a tab panel?"
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "passed"
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "passed"
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "needs discussion",
```

Comment:
I think this rule is failing because we don't allow a way to disable the animation that occurs on responsive tabs